pkg/apis/workflow/v1alpha1/workflow_types.go

```diff
@@ -677,6 +677,9 @@ type Parameter struct {
 	// GlobalName exports an output parameter to the global scope, making it available as
 	// '{{workflow.outputs.parameters.XXXX}} and in workflow.status.outputs.parameters
 	GlobalName string `json:"globalName,omitempty" protobuf:"bytes,5,opt,name=globalName"`
+
+	// Enum hold a list of values to choose from the actual value of the parameter
+	Enum []Int64OrString `json:"enum,omitempty" protobuf:"bytes,6,rep,name=enum"`
```

Comment:
Int64OrString is really just for keeping compatibility with old manifest. I think this can just be a strong.

Reply:
so during validation, I compare its value against `Value` variable, which is `Int64OrString`.
So, I thought its better to keep the same type.

Reply:
@alexec should I keep the same because of type of `Value` variable or change it ?

Reply:
no, i think you're correct to use it actually